projects/kit/components/input-date-multi/input-date-multi.module.ts

```diff
@@ -0,0 +1,43 @@
+import {CommonModule} from '@angular/common';
+import {NgModule} from '@angular/core';
+import {FormsModule} from '@angular/forms';
+import {MaskitoModule} from '@maskito/angular';
+import {TuiLetModule, TuiMapperPipeModule} from '@taiga-ui/cdk';
+import {
+    TuiCalendarModule,
+    TuiHostedDropdownModule,
+    TuiLinkModule,
+    TuiPrimitiveTextfieldModule,
+    TuiSvgModule,
+    TuiTextfieldComponent,
+    TuiTextfieldControllerModule,
+    TuiWrapperModule,
+} from '@taiga-ui/core';
+import {TuiInputTagModule} from '@taiga-ui/kit/components/input-tag';
+import {TuiValueAccessorModule} from '@taiga-ui/kit/directives';
+import {PolymorpheusModule} from '@tinkoff/ng-polymorpheus';
+
+import {TuiInputDateMultiComponent} from './input-date-multi.component';
+
+@NgModule({
+    imports: [
+        CommonModule,
+        MaskitoModule,
+        PolymorpheusModule,
+        TuiWrapperModule,
+        TuiCalendarModule,
+        TuiSvgModule,
+        TuiLinkModule,
+        TuiHostedDropdownModule,
+        TuiPrimitiveTextfieldModule,
+        TuiValueAccessorModule,
```

Comment:
Do we need this? I think it is only necessary for the old mask.

Reply:
fixed